### ISSUE TITLE
Content request: ETH issuance aka how many ETH are there?

### ISSUE THREAD

**Issue by konopkja** [ethereum/ethereum-org-website#13644]
### Is your feature request related to a problem? Please describe.

This is a content request for anyone to write a beginner friendly article about ETH supply.

Whoever wants to write the article, please first comment on this issue and we will assign it to you.

Use this style guide and best practices to format the article appropriately for our audience: https://ethereum.org/en/contributing/style-guide/

### Describe the solution you'd like

A beginner friendly article explaining ETH supply.

- How many ETH are there?
- How does the supply work?
- How is it different from real world currencies?
- What are the implications and benefits?

### Describe alternatives you've considered

Alternatively, in smaller size could live on https://ethereum.org/en/eth/ 

### Would you like to work on this issue?

- [ ] Yes
- [X] No

**Comment by panditdhamdhere** [comment]
Hey @konopkja can you assign this to me? I'd like to work on this.
Thank you.

**Comment by panditdhamdhere** [comment]
@konopkja Hi can I know where on which page I have to create PR.

**Comment by konopkja** [comment]
This would be a brand new page. - A subpage of https://ethereum.org/en/eth/ likely

**Comment by panditdhamdhere** [comment]
Yeah! But where I've to submit the content. Apologise!

Thank you

**Comment by konopkja** [comment]
not sure i follow, can you be more specific?

**Comment by pettinarip** [comment]
> Yeah! But where I've to submit the content. Apologise!
> 
> Thank you

Hey @panditdhamdhere, thanks for helping on this!

So, for this new page we would want it to live under https://ethereum.org/en/eth/supply

To do that, you will need to:
1. Create a new md file under https://github.com/ethereum/ethereum-org-website/blob/dev/public/content/eth/supply/index.md (you will need to create the folders as well)
2. Add the new path `"/eth/supply"` to the `temporalAllowedPages` array under https://github.com/ethereum/ethereum-org-website/blob/dev/src/lib/utils/md.ts#L30

Then, inside the new md file, you will need to add a bit of frontmatter and then the content:
```md
---
title: ETH supply
description: ...
lang: en
---

Content...
```

Thats all. LMK if you get stuck or need help!

**Comment by panditdhamdhere** [comment]
Wow! Thank you very much @pettinarip and @konopkja means alot! working on this

**Comment by konopkja** [comment]
Hello, upon internal review with the core team we decided to close linked PR as we need a more thorough page on issuance. We will likely reuse some of the content here and combine it with https://ethereum.org/en/roadmap/merge/issuance/#how-the-merge-impacts-eth-supply content that is more specific.

**Comment by Hedwika** [comment]
Hi I would like to cover this topic, first draft I may write until the end of November. Thank you!

**Comment by konopkja** [comment]
Hi @Hedwika  thank you for picking this up!

This article should be very low effort read for beginners with minimal technical language and web3 jargon. (= how would you explain eth issuance to your non crypto friends who dont even have wallet and only use exchanges?)

The structure makes mostly sense, just keep in mind that the article limit should be up to 1000 words. 

**Few concerns:**
- i dont think its necessary to speak about the history, should be only about the present and future
- the summary answer to "how many ETH tokens are there?" should be right at the top of the article
- so maybe the ETH Supply and Issuance Today should be first
- a lot of users come to as with request explaining ultrasound money so thats a good idea
- limit to just 5 links (internal and external) in the article

Tips on what language use in the article here: https://ethereum.org/en/contributing/style-guide/

**Comment by Hedwika** [comment]
I just created a PR based on the guidelines of @pettinarip from August 16: I needed to create a folder "eth", because it has not existed yet, inside it a "supply" folder and there an index.md

However, I haven't added the path yet, not sure where to put it to have it right. May you please help me with this?

Hope it was added well as I do it for the first time:
https://github.com/ethereum/ethereum-org-website/pull/14409

Looking forward to your ideas how to improve the article @konopkja thank you 🙏

**Comment by konopkja** [comment]
i think the path should be like /eth/supply

**Comment by Hedwika** [comment]
Agree, but not sure where into the array `temporalAllowedPages` to add it: There are topic like Use cases, Staking etc. (comments) with numbers of articles and the ticker emoji. I feel it might belongs under Static, shall I add it and change the number and delete the tick box? I know it's not a big deal, just do not want to create any mess :)

Also, do you have ny feedback to the article at this point? I did my best, but still see this as the first draft and I am ready to edit it to fit the Ethereum.org website content more.

**Comment by konopkja** [comment]
> Agree, but not sure where into the array `temporalAllowedPages` to add it: There are topic like Use cases, Staking etc. (comments) with numbers of articles and the ticker emoji. I feel it might belongs under Static, shall I add it and change the number and delete the tick box? I know it's not a big deal, just do not want to create any mess :)
> 
> Also, do you have ny feedback to the article at this point? I did my best, but still see this as the first draft and I am ready to edit it to fit the Ethereum.org website content more.

@wackerow  @pettinarip @corwintines  pls advise on the question

@Hedwika  i could not read the article as preview deploy is broken and it seems like your PR changed almost 2k files? i think this is mistake, it should not be rewriting everything, only modifying / adding few files

**Comment by corwintines** [comment]
Closing with above PR
